pkg/proto/configuration/jwt/jwt.proto

```diff
@@ -62,11 +63,25 @@ message AuthorizationHeaderParserConfiguration {
   //
   // More details about JMESPath can be found at https://jmespath.org/.
   //
+  // This JMESPath expression will be evaluated against a JSON object
+  // with the following structure:
+  //
+  //     {
+  //       "files": ...
+  //       "payload": ...
```

Comment:
Probably worth documenting this as:

```
"files": map<string, any>,
"payload": any
```

For consistency with the other places we use JMESPath.